README.md

```diff
@@ -1,4 +1,4 @@
-# Aerie Mission Model Template
+# Aerie Template
 
 This repo provides an Aerie mission model template, which is meant as a starting point for building a new mission model in Aerie.
```

Comment:
I think this part of the docs may need to be updated since mission model seems to be getting moved to a subproject